content/news/playtest-20151213.md

```diff
@@ -0,0 +1,15 @@
+We are slowly making progress towards fixing all the crashes and issues our testers (you!) have found, and the final release is imminent. Thanks to everyone who took the time to play the previous playtests and report any bugs!
+
+We have fixed all but one of the remaining issues, so it's time for another playtest. In particular, it contains the following fixes:
+
+* A GPS-related crash has been fixed (reported by Zypres and abdcdefg30, thanks to both!).
+* A crash that occured when producing an actor without any IOccupySpace traits in conjunction with UpgradeActorsNear has been fixed(reported by GraionDilach, thanks!).
+* Hardware cursors will not be permanently disabled when their creation fails (reported by pi99y, thanks!).
+* Using both mouse buttons in quick succession when attack-moving will work properly again (first reported by Aaron Lloyd, thanks!).
```

Comment:
This is making it sound buggier than it was. Can we call it what it was - "pressing both mouse keys at once produced unwanted results"?